src/atoms/prompt.jsx

```diff
@@ -71,45 +107,61 @@ class Prompt extends Component {
-                ref: (promptAction) => { this.promptAction = promptAction; },
-            });
-        }
-
-        if (children && children.type.name === 'Dropdown') {
-            return React.cloneElement(children, {
-                className: promptActionClasses,
-                buttonColor: show && children.props.buttonColor !== 'transparent' ? 'disable' : children.props.buttonColor,
-                onChange: this.onClick,
-                ref: (promptAction) => { this.promptAction = promptAction; },
-            });
-        }
 
-        return React.cloneElement(children, {
-            className: `${promptActionClasses} ${show ? 'color-static' : null}`,
-            onClick: this.onClick,
-            ref: (promptAction) => { this.promptAction = promptAction; },
+        this.setState({
+            inlineVerticalAlign: `${childHeight + negativeSpace}px`,
         });
     }
 
+    // renderAction() {
+    //     const { children } = this.props;
+    //     const { show } = this.state;
+
+    //     const childClasses = (children && children.props.className) || null;
+    //     const promptActionClasses = ClassNames('prompt-action', childClasses, {
+    //         'prompt-action-disable': show,
+    //     });
+    //     console.log('children', children);
+    //     console.log('children.type.name', children.type.name);
+
+    //     if (children && children.type.name === 'Button') {
+    //         return React.cloneElement(children, {
+    //             className: promptActionClasses,
+    //             color: show ? 'disable' : children.props.color,
+    //             onClick: this.onClick,
+    //             ref: (promptAction) => { this.promptAction = promptAction; },
+    //         });
+    //     }
+
+    //     if (children && children.type.name === 'DropdownButton)') {
+    //         return (
+    //             <div ref={(ref) => { this.promptAction = ref; }}>
+    //                 {React.cloneElement(children, {
+    //                     className: promptActionClasses,
+    //                     color: show && children.props.buttonColor !== 'transparent' ? 'disable' : children.props.buttonColor,
+    //                 })}
+    //             </div>
+    //         );
+    //     }
+
+    //     if (children && children.type.name === 'Dropdown') {
+    //         return React.cloneElement(children, {
+    //             className: promptActionClasses,
+    //             buttonColor: show && children.props.buttonColor !== 'transparent' ? 'disable' : children.props.buttonColor,
+    //             onChange: this.onClick,
+    //             ref: (promptAction) => { this.promptAction = promptAction; },
+    //         });
+    //     }
+
+    //     return React.cloneElement(children, {
+    //         className: `${promptActionClasses} ${show ? 'color-static' : null}`,
+    //         onClick: this.onClick,
+    //         ref: (promptAction) => { this.promptAction = promptAction; },
+    //     });
+    // }
```

Comment:
Can we please remove the commented code, once we commit to the current implementation?